src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const amount = days * 40;
+  const basicDiscount = 50;
+  const smallDiscount = 20;
+
+  return days >= 7
+    ? amount - basicDiscount : days >= 3 && days < 7
+      ? amount - smallDiscount : amount;
 }
```

Comment:
Let's move the days count to variables. They could be changed too.
And also don't nest ternary operators into ternary operators. It will be better to use if(){} statement here.